frontend/src/metabase/palette/shortcuts/global.ts

```diff
@@ -0,0 +1,61 @@
+import { t } from "ttag";
+
+import type { ShortcutDef } from "../types";
+export const globalShortcuts: Record<string, ShortcutDef> = {
+  "create-question": {
+    name: "Create a question",
+    shortcut: ["q"],
+    shortcutGroup: "global",
+  },
+  "create-native-query": {
+    name: "Create a native query",
+    shortcut: ["n"],
+    shortcutGroup: "global",
+  },
+  "create-dashboard": {
+    name: "Create a dashboard",
+    shortcut: ["d"],
+    shortcutGroup: "global",
+  },
+  "create-collection": {
+    name: "Create a collection",
+    shortcut: ["c"],
+    shortcutGroup: "global",
+  },
+  "create-model": {
+    name: "Create a model",
+    shortcut: ["m"],
+    shortcutGroup: "global",
+  },
```

Comment:
nit, but the default behavior for this is to close the sidebar, but this leaves it open

![Screenshot 2025-04-01 at 12 16 02 PM](https://github.com/user-attachments/assets/e5fbbfe2-ebfc-4443-a6bc-6445bda47f27)